docs/content/docs/install/manual_demo/_index.md

```diff
@@ -126,8 +126,25 @@ A Kubernetes Service, Deployment, and Service Account for applications `bookbuye
 To view these resources on your cluster, run the following commands:
 
 ```bash
-kubectl get svc --all-namespaces
-kubectl get deployment --all-namespaces
+kubectl get deployments -n bookbuyer
+kubectl get deployments -n bookthief
+kubectl get deployments -n bookstore
+kubectl get deployments -n bookwarehouse
+
+kubectl get pods -n bookbuyer
+kubectl get pods -n bookthief
+kubectl get pods -n bookstore
+kubectl get pods -n bookwarehouse
+
+kubectl get services -n bookbuyer
+kubectl get services -n bookthief
+kubectl get services -n bookstore
+kubectl get services -n bookwarehouse
+
+kubectl get endpoints -n bookbuyer
+kubectl get endpoints -n bookthief
+kubectl get endpoints -n bookstore
+kubectl get endpoints -n bookwarehouse
```

Comment:
Don't think this is too relevant from the perspective of the manual demo. I'd keep this minimal, Deployments, Services, Pods should suffice.